### PR TITLE
Major refactoring

### PR DESCRIPTION
**Pull request by Progi1984**
Based on the work of @pablopolyte

### CODE REVIEW

File: sql/install.php [hunk 1]
```diff
@@ -76,4 +73,4 @@
     }
 }
 
-return true;
+return true;
```
Comment (matks) [inline sql/install.php]: Missing ending whiteline

File: views/css/reassurance.css [hunk 2]
```diff
@@ -20,3 +33,7 @@
     padding-top: 5px;
     margin-right: 5px;
 }
+
+.item-product img {
+    width:35px;
+}
```
Comment (matks) [inline views/css/reassurance.css]: Missing ending empty line

File: views/js/back.js [hunk 3]
```diff
@@ -39,65 +39,28 @@ $(window).ready(function () {
...
+  }
 
-  $(document).on('change', 'input[name="PSR_REDIRECTION_3"]', (e) => {
+  $(document).on('change', 'input[name="PSR_REDIRECTION_1"],input[name="PSR_REDIRECTION_2"],input[name="PSR_REDIRECTION_3"]', (e) => {
     if ($(e.target).val() == 0) {
-      $('.psr-cms').removeClass('active');
-      $('.psr-url').removeClass('active');
-      $('.psr-cms').addClass('inactive');
-      $('.psr-url').addClass('inactive');
+      setEnabledPSR('cms', false);
```
Comment (matks) [inline views/js/back.js]: 👍

File: views/js/menu.js [hunk 4]
```diff
@@ -153,18 +141,15 @@ $(window).ready(function () {
...
-        } else {
-          dataToSave[lang][type] = '';
-        }
+      } else if (typeof($('input', elem).val()) != 'undefined') {
+        dataToSave[lang][type] = $('input', elem).val();
       }
     });
 
-    formData = new FormData();
+    var formData = new FormData();
```
Comment (matks) [inline views/js/menu.js]: 👍

File: views/templates/admin/tabs/content/config_elements/language.tpl [hunk 5]
```diff
@@ -23,29 +23,27 @@
...
+                        <option {if $lang.id_lang == $defaultFormLanguage}selected="selected"{/if}
+                                value="{$lang.id_lang}">{$lang.name|escape:'htmlall':'UTF-8'}</option>
+                    {/foreach}
+                </select>
+            </div>
         </div>
     </div>
     <div class="clearfix"></div>
 </div>
+{/if}
```
Comment (matks) [inline views/templates/admin/tabs/content/config_elements/language.tpl]: Missing ending empty line

File: views/templates/admin/tabs/content/config_elements/title.tpl [hunk 6]
```diff
@@ -37,12 +37,11 @@
             <div class="col-xs-12 col-sm-12 col-md-7 col-lg-12">
                 <input type="text" name="title-{$language.id_lang|escape:'htmlall':'UTF-8'}" max="100"
                        class="form-control"
-                       value="{$allblockByShop[{$language.id_lang}][$aBlock['id_psreassurance']]['title']}">
+                       value="{$allblockByShop[{$language.id_lang}][$aBlock['id_psreassurance']]['title']|escape:'htmlall':'UTF-8'}">
```
Comment (matks) [inline views/templates/admin/tabs/content/config_elements/title.tpl]: 👍

File: sql/install.php (hunk 1 above)
Comment (Progi1984) [reply sql/install.php]: Fixed

File: views/css/reassurance.css (hunk 2 above)
Comment (Progi1984) [reply views/css/reassurance.css]: Fixed

File: views/js/back.js (hunk 3 above)
Comment (Progi1984) [reply views/js/back.js]: :smile:

File: views/js/menu.js (hunk 4 above)
Comment (Progi1984) [reply views/js/menu.js]: :)

File: views/templates/admin/tabs/content/config_elements/language.tpl (hunk 5 above)
Comment (Progi1984) [reply views/templates/admin/tabs/content/config_elements/language.tpl]: Fixed

File: views/templates/admin/tabs/content/config_elements/title.tpl (hunk 6 above)
Comment (Progi1984) [reply views/templates/admin/tabs/content/config_elements/title.tpl]: :smile:

**Comment by matks** [comment]
Thank you @Progi1984

**Pull request merged**